docs/core/tools/dotnet-add-reference.md

```diff
@@ -19,14 +20,11 @@ ms.assetid: 5e2a3efd-443c-4f23-a1b1-a662a5387879
 
 ## Synopsis
 
-```
-dotnet add [project] reference [-f|--framework] <project_references>
-dotnet add reference [-h|--help]
-```
+`dotnet add [<PROJECT>] reference [-f|--framework <FRAMEWORK>] <PROJECT_REFERENCES> [-h|--help]`
```

Comment:
I'd prefer not to add the options parameters in the synopsis. This could get long. And same comment for PROJECT as in the previous command.